### PR TITLE
Code Util Upgrade - Improved version and timeouts

### PR DESCRIPTION
**Pull request by vinodmuralidharan01**
Some improvements have been made
1. Added timeout for the code server setup (The process would wait for 35 min during On creation and 20 min during On Start)
2. Disabled nbserver proxy serverextension
3. VS Code version been upgraded from 4.16.1 to 4.91.1

### CODE REVIEW

**Review by svyotov — DISMISSED** [review]

File: code-server/notebook-instances/samples/vscode-with-glue/on-create.sh [hunk 1]
```diff
@@ -3,15 +3,27 @@
...
+
 # Check for glue_ready file and wait for code_server_setup.py if needed
 if [ -e /home/ec2-user/glue_ready ]; then
-    wait $code_server_setup_pid
+    while kill -0 $code_server_setup_pid 2>/dev/null; do
+      if [ $elapsed_time -ge $timeout_duration ]; then
+          echo "Process $code_server_setup_pid did not complete within 12 minutes."
+          exit 1
+      fi
+    sleep $check_interval
```
Comment (svyotov) [inline code-server/notebook-instances/samples/vscode-with-glue/on-create.sh:25]: the indentation if off.

Comment (vinodmuralidharan01) [reply code-server/notebook-instances/samples/vscode-with-glue/on-create.sh:25]: indentation corrected

File: code-server/notebook-instances/samples/vscode-with-glue/on-start.sh [hunk 2]
```diff
@@ -37,7 +49,11 @@ echo "INFO: Site package directory is --- $SITE_PACKAGES_DIR"
 
 # Install Glue Sessions to Env
 echo "INFO: Installing AWS Glue Sessions with pip"
-pip install aws-glue-sessions==1.0.0
+# Get Latest Glue Session version
+#LATEST_GLUE_SESSIONS=$(curl -s https://pypi.org/pypi/aws-glue-sessions/json | python -c "import sys, json; print(json.load(sys.stdin)['info']['version'])")
```
Comment (svyotov) [inline code-server/notebook-instances/samples/vscode-with-glue/on-start.sh]:
```suggestion
# LATEST_GLUE_SESSIONS=$(curl -s https://pypi.org/pypi/aws-glue-sessions/json | python -c "import sys, json; print(json.load(sys.stdin)['info']['version'])")
```

File: code-server/notebook-instances/samples/vscode-with-glue/on-start.sh [hunk 3]
```diff
@@ -37,7 +49,11 @@ echo "INFO: Site package directory is --- $SITE_PACKAGES_DIR"
 
 # Install Glue Sessions to Env
 echo "INFO: Installing AWS Glue Sessions with pip"
-pip install aws-glue-sessions==1.0.0
+# Get Latest Glue Session version
+# LATEST_GLUE_SESSIONS=$(curl -s https://pypi.org/pypi/aws-glue-sessions/json | python -c "import sys, json; print(json.load(sys.stdin)['info']['version'])")
+#pip install aws-glue-sessions==$LATEST_GLUE_SESSIONS
```
Comment (svyotov) [inline code-server/notebook-instances/samples/vscode-with-glue/on-start.sh]:
```suggestion
# pip install aws-glue-sessions==$LATEST_GLUE_SESSIONS
```

**Review by svyotov — DISMISSED** [review]

File: code-server/notebook-instances/samples/vscode-with-glue/on-create.sh [hunk 4]
```diff
@@ -1,22 +1,37 @@
 #!/bin/bash
-# On-Create Script
+# On-Start Script
```
Comment (svyotov) [inline code-server/notebook-instances/samples/vscode-with-glue/on-create.sh]: nope

**Review by svyotov — APPROVED** [review]

**Pull request merged**